ampd/src/handlers/multisig.rs

```diff
@@ -278,6 +275,44 @@ mod test {
         .unwrap()
     }
 
+    // this returns an event that is named SigningStarted,
+    // but the fields are different than what the handler expects
+    fn wrong_signing_started_event() -> events::Event {
+        let pub_keys = (0..10)
+            .map(|_| (rand_account().to_string(), rand_public_key()))
+            .collect::<HashMap<String, PublicKey>>();
+
+        let poll_started = SigningStarted {
+            session_id: Uint64::one(),
+            worker_set_id: "worker_set_id".to_string(),
+            pub_keys,
+            msg: MsgToSign::unchecked(rand_message()),
+            chain_name: rand_chain_name(),
+            expires_at: 100u64,
+        };
+
+        let mut event: cosmwasm_std::Event = poll_started.into();
+        event.ty = format!("wasm-{}", event.ty);
```

Comment:
why is this not part of the type conversion?

Reply:
Not sure. Somewhere along the way this is added automatically, but not totally sure at what step.